### ISSUE TITLE
Type of Ivy repository

### ISSUE THREAD

**Issue by rangzen** [ysb33r/ivypot-gradle-plugin#14]
Hello, 

With Gradle 2.12 and IvyPot 0.3.7, unlike in https://github.com/ysb33r/ivypot-gradle-plugin#usage I have to specify the Ivy type to use it, like that :

```
repositories {
  ivy {
    url '/path/to/repo'
    layout 'ivy'
  }
}
```

Does IvyPot use an old default value?
Does the documentation need to be updated?

Thank you.

**Comment by ysb33r** [comment]
I did a fix (#11) in the 0.3.7 which allows a script author to specify the kind of layout to create. Currently the default layout is ivy.  I am not sure whether Gradle 2.12 now enforces one to specifiy the layout to create. 

I would howver recommend that you tell `syncRemoteRepositories` to use the `gradle` layout instead. (In the 0.4 release I plan to make that the default).

**Comment by ysb33r** [comment]
I'll push some notes to the README for now.
